k8s/autopilot/autopilot.go

```diff
@@ -0,0 +1,123 @@
+package autopilot
+
+import (
+	"fmt"
+	"os"
+
+	autopilotclientset "github.com/libopenstorage/autopilot-api/pkg/client/clientset/versioned"
+	"k8s.io/client-go/rest"
+	"k8s.io/client-go/tools/clientcmd"
+)
+
+var (
```

Comment:
I am curious about that singleton. If you ask for instance it will have config and autopilot field null. 
Maybe you should initialize it with sync.Once to return configured client object.

Reply:
Good point. Should have something like this in the Instance() method.
```
func Instance() {
   once.Do(func() {
     instance = &Client{}
   })
   return instance
}
```

Reply:
@stgleb initialization is done lazily by initClient() method. [comment](https://github.com/portworx/sched-ops/pull/173#discussion_r365386365)

@piyush-nimbalkar there is no need for the once.Do functionc as instance object is defined not as a pointer and is initialized as a zero value.

Reply:
@saheienko I understand for instance as a value, but what about instance fields?

Reply:
@stgleb https://github.com/portworx/sched-ops/pull/173#discussion_r365386365
There is `initClient` invoked for each method that ensures client initialization. The same implementation approach is used for existing version of sched-ops.